### PR TITLE
Location select 0

### PR DESCRIPTION
**Pull request by zachpren0**
Location Selection works on live orders. changes location update to selected location on selection. live order display also only shows orders from selected location.

### CODE REVIEW

**Review by camwlsn — APPROVED** [review]
Great work implementing this search feature

**Review by SpeiserK — APPROVED** [review]
Nice work. Great addition to employee functionality.

File: MERN Scripts/mern-emsystem/src/components/LocationSelect.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    
+    <select value={selectedLocation} onChange={handleChange} >
+      <option value="Rutland">Select a location</option>
+      <option value="West Kelowna">West Kelowna</option>
+      <option value="Rutland">Rutland</option>
+      <option value="Mission">Mission</option>
+      <option value="Lake Country">Lake Country</option>
+      <option value="Glenmore"> Glenmore </option>
+      <option value="Kelowna Central"> Kelowna Central </option>
+    </select>
```
Comment (amurph24) [inline MERN Scripts/mern-emsystem/src/components/LocationSelect.js:23]: we should retrieve this from the Location Collection in the DB, can we do that before merging? i.e. map the Location.Name values for the options

File: MERN Scripts/mern-emsystem/src/pages/Emp.js [hunk 2]
```diff
@@ -22,15 +23,11 @@ function Emp(){
         </div>
         <body className="appContainer">
             <div className="empLeftDiv"> 
+                <LocationLiveDisp queryLoc={empLocation} />
                 <h2>Orders that need pickup</h2>
-                 <OrderLiveDisp query1={'Approved'} />
+                 <OrderLiveDisp query1={'Approved'} queryLoc={empLocation} />
```
Comment (amurph24) [inline MERN Scripts/mern-emsystem/src/pages/Emp.js:28]: 👏

File: MERN Scripts/mern-emsystem/src/components/LocationSelect.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  const handleChange = (event) => {
+    setSelectedLocation(event.target.value);
+    localStorage.setItem('selectedLocation', event.target.value);
+    window.location.reload();
+  }
+
+  return (
+    
+    <select value={selectedLocation} onChange={handleChange} >
+      <option value="Rutland">Select a location</option>
```
Comment (amurph24) [inline MERN Scripts/mern-emsystem/src/components/LocationSelect.js:16]: probably irrelevant but should the value here be some default or null value?

**Review by amurph24 — CHANGES_REQUESTED** [review]

**Review by amurph24 — APPROVED** [review]

File: MERN Scripts/mern-emsystem/src/components/LocationSelect.js (hunk 1 above)
Comment (zachpren0) [reply MERN Scripts/mern-emsystem/src/components/LocationSelect.js:23]: I will make an issue for this , will make another pr for this

**Pull request merged**